airbyte-integrations/connectors/destination-azure-blob-storage/src/main/java/io/airbyte/integrations/destination/azure_blob_storage/csv/AzureBlobStorageCsvWriter.java

```diff
@@ -46,17 +46,9 @@ public AzureBlobStorageCsvWriter(final AzureBlobStorageDestinationConfig config,
 
     this.blobOutputStream = new BufferedOutputStream(appendBlobClient.getBlobOutputStream(), config.getOutputStreamBufferSize());
 
-    if (isNewlyCreatedBlob) {
-      this.csvPrinter = new CSVPrinter(
-          new PrintWriter(blobOutputStream, false, StandardCharsets.UTF_8),
-          CSVFormat.DEFAULT.withQuoteMode(QuoteMode.ALL)
-              .withHeader(csvSheetGenerator.getHeaderRow().toArray(new String[0])));
-    } else {
-      // no header required for append
-      this.csvPrinter = new CSVPrinter(
-          new PrintWriter(blobOutputStream, false, StandardCharsets.UTF_8),
-          CSVFormat.DEFAULT.withQuoteMode(QuoteMode.ALL));
-    }
+    this.csvPrinter = new CSVPrinter(new PrintWriter(blobOutputStream, false, StandardCharsets.UTF_8),
```

Comment:
[OPTIONAL] Optional suggestion would be here to put this false statement to some boolean variable. This will help in future ti understand what is responsibility of this variable. Especially in reviews.

Reply:
updated it